test/cipherlist_test.c

```diff
@@ -215,9 +215,41 @@ static int test_default_cipherlist_explicit(void)
     return result;
 }
 
+/* SSL_CTX_set_cipher_list() should fail if it clears all TLSv1.2 ciphers. */
+static int test_default_cipherlist_clear(void)
+{
+    SETUP_CIPHERLIST_TEST_FIXTURE();
+    SSL* s;
+    if (fixture == NULL)
+        return 0;
+
+    result = TEST_int_eq(SSL_CTX_set_cipher_list(fixture->server,
+                "no-such-cipher"), 0);
+    if (!result) goto end;
```

Comment:
All these `if (!result) goto end;` should be on two lines. Actually, all these tests can be combined into a single if statement.